cli/auth.go

```diff
@@ -256,6 +256,9 @@ func (a *authFlow) waitForUser(ctx context.Context, code *deviceCodeResponse, di
 		} else if err == nil {
 			return resp, nil
 		}
+		if err = res.Body.Close(); err != nil {
```

Comment:
we don't care about an error here, so you should use UncheckedError(func() { res.Body.Close for both but in a defer after the .Do is done and the first error has been checked